native_client/alphabet.h

```diff
@@ -52,19 +53,31 @@ class Alphabet {
     }
   }
 
-  size_t GetSize() {
+  size_t GetSize() const {
     return size_;
   }
 
   bool IsSpace(unsigned int label) const {
-    //TODO: we should probably do something more i18n-aware here
-    const std::string& str = StringFromLabel(label);
-    return str.size() == 1 && str[0] == ' ';
+    return label == space_label_;
+  }
+
+  unsigned int GetSpaceLabel() const {
+    return space_label_;
+  }
+
+  template <typename T>
+  std::string LabelsToString(const std::vector<T>& input) const {
```

Comment:
Because ctcdecode uses int for the labels, not unsigned int. For this function, any integral type will do, it just needs to be comparable with the type stored in `Alphabet`.